### PR TITLE
 #12 Added the ability to save nonogram

### PR DESCRIPTION
**Pull request by maik-nack**

### CODE REVIEW

File: app/src/main/java/com/mithridat/nonoconverter/Utils.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+     * @return width of the text
+     */
+    public static int getWidthText(Paint paint, String text) {
+        Rect bounds = new Rect();
+        paint.getTextBounds(text, 0, text.length(), bounds);
+        return bounds.left + bounds.width();
+    }
+
+    /**
+     * Method for getting width of the text by specific paint
```
Comment (MaximSmolskiy) [inline app/src/main/java/com/mithridat/nonoconverter/Utils.java]: Please correct comment: width -> height

File: app/src/main/java/com/mithridat/nonoconverter/Utils.java [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+     *
+     * @param paint - the Paint for drawing
+     * @param text - the text height width is unknown
```
Comment (MaximSmolskiy) [inline app/src/main/java/com/mithridat/nonoconverter/Utils.java]: Please correct comment: height width -> which height

**Review by MaximSmolskiy — APPROVED** [review]
Looks great!

**Review by NikitaSukhanov — APPROVED** [review]
Good job!

**Pull request merged**